src/main/options-normalizer.js

```diff
@@ -33,8 +32,8 @@ class FlagSchema extends vnopts.ChoiceSchema {
       if (suggestion) {
         utils.logger.warn(
           [
-            `Unknown flag ${chalk.yellow(utils.descriptor.value(value))},`,
-            `did you mean ${chalk.blue(utils.descriptor.value(suggestion))}?`,
+            `Unknown flag ${utils.descriptor.value(value)},`,
```

Comment:
We can pass `chalk` through `normalizeCliOptions`.

Reply:
FYI: I tried to move `normalizeCliOptions` to `src/cli/` before, but failed.

Reply:
Ah I was about to work on this, but then I spotted #12162. Closing this PR in favor of yours, thanks!